root/static/scripts/tests/Control/URLCleanup.js

```diff
@@ -2701,6 +2701,26 @@ limited_link_type_combinations: [
             expected_clean_url: 'https://kashinavi.com/song_view.html?68574',
        only_valid_entity_types: ['work'],
   },
+  // KBR
+  {
+                     input_url: 'http://opac.kbr.be/LIBRARY/doc/AUTHORITY/14160974',
+             input_entity_type: 'artist',
+    expected_relationship_type: 'otherdatabases',
+            expected_clean_url: 'https://opac.kbr.be/LIBRARY/doc/AUTHORITY/14160974',
+       only_valid_entity_types: ['artist', 'label'],
+  },
+  {
+                     input_url: 'https://opac.kbr.be/LIBRARY/doc/AUTHORITY/13974166',
+             input_entity_type: 'label',
+    expected_relationship_type: 'otherdatabases',
+       only_valid_entity_types: ['artist', 'label'],
+  },
+  {
+                     input_url: 'https://opac.kbr.be/LIBRARY/doc/SYRACUSE/17060572',
+             input_entity_type: 'release',
+    expected_relationship_type: 'otherdatabases',
+       only_valid_entity_types: ['release'],
+  },
```

Comment:
There is a `clean` function so it seems worth testing it didn’t make any unwanted change.

Reply:
Hmm, I thought you had told me in the past that we shouldn't always use `expected_clean_url` because it's redundant, but maybe I misremembered :) Ok, added!

Reply:
Right, it is redundant if it is the same input URL. It isn’t if it is a different URL pattern.

Maybe it could be added to the current documentation for this test field: https://github.com/metabrainz/musicbrainz-server/blob/v-2022-09-06/root/static/scripts/tests/Control/URLCleanup.js#L85-L90